### PR TITLE
doc,bgpv1: Add documentation about the address family option

### PR DESCRIPTION
**Pull request by YutaroHayakawa**
Add documentation about the address family option and caveats about the graceful restart address family mentioned in https://github.com/cilium/cilium/issues/30367/.

```release-note
doc,bgpv1: Add documentation about the address family option
```

### CODE REVIEW

File: Documentation/network/bgp-control-plane.rst [hunk 1]
```diff
@@ -691,3 +698,40 @@ Once configured, the additional Path Attributes advertised with the routes for a
...
+Address Families
+^^^^^^^^^^^^^^^^
+
+By default, the BGP Control Plane advertises IPv4 Unicast and IPv6 Unicast
+Multiprotocol Extensions Capability (`RFC-4760`_) as well as Graceful Restart
+address families (`RFC-4724`_) if enabled. If you wish to change the default
+behavior and advertise only specific address families, you can use the
+``families`` field. The ``families`` field is a list of AFI (Address Family
+Identifier) and SAFI (Subsequent Address Family Identifier) pairs. The only
+option currently supported are ``{afi: ipv4, safi: unicast}`` and ``{afi: ipv6,
```
Comment (rastislavs) [inline Documentation/network/bgp-control-plane.rst]:
```suggestion
options currently supported are ``{afi: ipv4, safi: unicast}`` and ``{afi: ipv6,
```

File: Documentation/network/bgp-control-plane.rst [hunk 2]
```diff
@@ -585,9 +585,16 @@ Configure graceful restart on per-neighbor basis, as follows:
...
 
-.. note::
+.. warning::
 
-   When enabled, graceful restart capability is advertised for IPv4 and IPv6 address families.
+   When enabled, graceful restart capability is advertised for IPv4 and IPv6
+   address families by default. From v1.15, we have a known issue where Cilium
+   takes long time (approximately 300s) to restart route advertisement after
+   graceful restart when Cilium advertises both IPv4 and IPv6 address families,
+   but remote only advertises one of them. You can work around this issue by
```
Comment (rastislavs) [inline Documentation/network/bgp-control-plane.rst]:
```suggestion
   but remote peer advertises only one of them. You can work around this issue by
```

**Review by rastislavs — APPROVED** [review]
LGTM, just some nits

File: Documentation/network/bgp-control-plane.rst [hunk 3]
```diff
@@ -585,9 +585,16 @@ Configure graceful restart on per-neighbor basis, as follows:
...
 
-.. note::
+.. warning::
 
-   When enabled, graceful restart capability is advertised for IPv4 and IPv6 address families.
+   When enabled, graceful restart capability is advertised for IPv4 and IPv6
+   address families by default. From v1.15, we have a known issue where Cilium
+   takes long time (approximately 300s) to restart route advertisement after
+   graceful restart when Cilium advertises both IPv4 and IPv6 address families,
+   but remote peer advertises only one of them. You can work around this issue
```
Comment (learnitall) [inline Documentation/network/bgp-control-plane.rst]:
```suggestion
   but a remote peer advertises only one of them. You can work around this issue
```

**Review by learnitall — APPROVED** [review]
Optional grammar suggestion, but otherwise LGTM. Thanks!

**Pull request merged**